tests/forms_tests/field_tests/test_disabledfield.py

```diff
@@ -0,0 +1,33 @@
+from django.forms import Field, Form
+from django.test import SimpleTestCase
+
+
+class TestForm(Form):
+    disabled_empty_initial = Field(disabled=True, required=False)
+    disabled_initial = Field(disabled=True, initial='Test')
+
+
+class DisabledFieldTest(SimpleTestCase):
+    def test_disabled_field(self):
```

Comment:
I'm more inclined to put the test in `test_base.py` rather than create a new file for behavior specific tests. The tests look like they're organized by class here.